detox/test/scripts/__snapshots__/verify_artifacts_are_not_missing.ios.test.js.snap

```diff
@@ -4,10 +4,10 @@ exports[` 1`] = `
 Array [
```

Comment:
We stopped testing artifacts file snapshots, no need to change it. PR might need a rebase/merge from master.

Reply:
Ok removing it + unupdating the doc :)

Reply:
Still, shouldn't we remove this snapshot in this case ? if so I can do it in this PR :)